Svg.Editor.Core/Tools/MoveTool.cs

```diff
@@ -51,7 +51,7 @@ protected override async Task OnDrag(DragGesture drag)
                     // or the active tool is PinTool
                     // otherwise we'd move and pan at the same time, yielding confusing results... :)
                     if ((Canvas.ActiveTool.ToolType != ToolType.Select) &&
-                        (Canvas.ActiveTool.GetType() != typeof(PinTool))) return;
+                        !(Canvas.ActiveTool is ISupportTextColor)) return;
```

Comment:
Is the pintool a "tooltype.select"?

Reply:
and: IsupportTextColor has nothing to do with moving ;-)
So please use yet another interface